cucumber/api/features/support/conjur_token.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
please use the one that is already implemented

Reply:
Yes there are two already implemented classes such as ConjurToken. But there are in other cucumber profiles' support directory - we can move this into the code itself or find a shared support folder to all cucumber tests

Reply:
see other comment on how to load these files

Reply:
fixed